solutions/avnm-secured-hub-and-spoke/README.md

```diff
@@ -15,44 +15,54 @@ urlFragment: virtual-network-manager-secured-hub-and-spoke
 
 This sample deploys Azure virtual networks in a hub and spoke configuration, using Azure Virtual Network Manager to manage Virtual Network connectivity and implement sample Security Admin Rules. A VPN Gateway and test VMs are deployed to complete the hub and spoke features.
 
-
 ## Deploy sample
 
-### Step 1: Create a Resource Group for the sample resources
+### Step 1: Environment
+
+```bash
+RESOURCEGROUP_NAME=rg-hub-spoke-eastus
+
+# Ensure the feature is enable
+az feature register --namespace "Microsoft.Compute" --name "EncryptionAtHost"
```

Comment:
it is safe to remove encryption at host as well and then remove step1 just hardcoding the name of hub rg where required.